src/app/content/actions.ts

```diff
@@ -22,3 +22,6 @@ export const receivePage = createStandardAction('Content/receivePage')<
 
 export const closeCallToActionPopup = createStandardAction('Content/closeCallToActionPopup')();
 export const openCallToActionPopup = createStandardAction('Content/openCallToActionPopup')();
+
+export const openNudgeStudyTools = createStandardAction('Content/openNudgeStudyTools')();
+export const closeNudgeStudyTools = createStandardAction('Content/closeNudgeStudyTools')();
```

Comment:
Previously ctaPopup actions were here. What would be a better place? Please take a note that `nudge` will be opened only if a book has studyGuides and this will be checked only after `content/hooks/locationChange`

Reply:
I imagine it could just go to `/studyGuides/actions.ts`. Login cta didn't depend on anything related to content, right?

Reply:
yea if this is study guide specific then i would put it in the /studyGuide section of logic/files

Reply:
actually going to defer on this until we implement the other nudge because i'm sure it'll become entangled